src/applications/check-in/travel-claim/pages/travel-intro/index.jsx

```diff
@@ -0,0 +1,116 @@
+import React, { useCallback } from 'react';
+import PropTypes from 'prop-types';
+import { useTranslation, Trans } from 'react-i18next';
+import { recordEvent } from '@department-of-veterans-affairs/platform-monitoring/exports';
+
+import Wrapper from '../../../components/layout/Wrapper';
+import { createAnalyticsSlug } from '../../../utils/analytics';
+import { hasMultipleFacilities } from '../../../utils/appointment';
+import { useFormRouting } from '../../../hooks/useFormRouting';
+import TravelEligibilityAddtionalInfo from '../../../components/TravelEligibilityAdditionalInfo';
+
+// Appointments will come from redux this is temp
+import { singleFacility } from './testAppointments';
+
+const TravelIntro = props => {
+  const { router, appointments = singleFacility } = props;
+  const { t } = useTranslation();
+
+  const { jumpToPage } = useFormRouting(router);
+  // @TODO logic need to figure out if we should go to multi-facility page or single
```

Comment:
Is this todo still valid? It looks the work is done unless I'm missing something.

Reply:
Oh yeah I think I forgot that one. But there is still work to do here, we need to mix this with the localStorage bit to determine total number of valid facilities and it also needs to get refactored to select the appts from redux. So are fine leaving it for now?